src/Robo/Commands/Blt/AliasCommand.php

```diff
@@ -166,10 +166,13 @@ protected function getClosingBracketPosition($contents, $start_pos) {
   }
 
   /**
-   * Creates a ~/.bash_profile on OSX if one does not exist.
+   * Creates a ~/.bash_profile on supporting systems if one does not exist.
+   *
+   * @throws \Acquia\Blt\Robo\Exceptions\BltException
    */
-  protected function createOsxBashProfile() {
-    if ($this->getInspector()->isOsx()) {
+  protected function createBashProfile() {
+    $inspector = $this->getInspector();
+    if ($inspector->isOsx() || $inspector->isAhEnv()) {
```

Comment:
As a Linux user, am I being excluded from this condition? IIRC the alias is being set up just fine when I run `blt blt:init:shell-alias` FYI.

Reply:
Hey @anavarre , this conditional does exclude Linux users (like you and me). However, that generally shouldn't be a problem because most Linux distros include at least a stub .bashrc or .bash_profile that BLT will happily modify, making this function unnecessary. Mac OS doesn't typically have one of these files OOTB, so BLT has to create it.